### ISSUE TITLE
While adding new VD Homey app crashes

### ISSUE THREAD

**Issue by PeterKawa** [ArjanKranenburg/virtual-devices#140]
Tested with Experimental, crash report ID:
41633d8d-0dc0-48bd-8b4d-0bb208c72b69
And tested afterwards with official release v0.9.5
While adding a VD, Homey app crashes. It happens when the screen us being loaded where you can enter the name. 
I killed Homey app, tried again, but no difference.
I think I'm on Homey Beta app, version = v.6.4.0 (also sent Android crash report to Athom)
Existing VD's work fine!

Cheers, and I bought Arjan a beer.
Peter

**Comment by ArjanKranenburg** [comment]
First of all: Thank you very much for the beer!
And thank you for this report. 
I’ve quickly tried to reproduce this, but the device is added OK for me. Are there any other details that could be of interest?
What Homey version (not the app, but Homey itself) do you use?
Do you try to add a device or a mode?
Does it also crash with iOS (if you have that)?
Do devices of other apps install OK?
Anything else?

with kind regards,

Arjan

**Comment by PeterKawa** [comment]
You're welcome, Arjan!
The beer did the trick, I just added 4 V devices and 4 V modusses.
Additional Info (may come in handy maybe)
- Homey v5.0.2
- Tried to add both Devices & Modusses
- I've got zero Apple stuff, so could not test
- I can't recall if i tried to install other devices, sorry.

Well, b/c it installs fine now I suggest closing this issue, Arjan.

Cheers and thanx for your work and time
Peter
